lib/ansible/plugins/connection/winrm.py

```diff
@@ -367,7 +376,7 @@ def _winrm_connect(self):
 
         winrm_host = self._winrm_host
         if HAS_IPADDRESS:
-            display.vvvv("checking if winrm_host %s is an IPv6 address" % winrm_host)
+            display.debug("checking if winrm_host %s is an IPv6 address" % winrm_host)
```

Comment:
Keep as before

Reply:
This is really a debug-statement, I don't see a point in showing this for every connection.